docs/contributors/getting-started.md

```diff
@@ -186,36 +186,30 @@ These logs contain information that may be relevant for regular users.
 
 ### JSON-RPC trace
 
-To see trace of incoming/outgoing JSON communication with the text editor or
-build server, create empty files in your machine cache directory.
+To see the trace of incoming/outgoing JSON communication with the text editor or build server, create empty files in `$WORKSPACE/.metals/` or your machine cache directory.  
```

Comment:
Could we mention that machine cache is not recommened, since multiple servers can override it? Also, we no longer mention when the cache is located, so we should probably add a section about it.